code/drasil-example/Drasil/DblPendulum/GenDefs.hs

```diff
@@ -29,7 +29,7 @@ genDefns = [velocityIXGD, velocityIYGD, accelerationIXGD, accelerationIYGD,
 
 -- ----------
 velocityIXGD :: GenDefn
-velocityIXGD = gdNoRefs velocityIXRC (getUnit velocity)
+velocityIXGD = gdNoRefs (OthModel velocityIXRC) (getUnit velocity)
```

Comment:
So, of course, for a lot of these, we'll want to go to a "better" model that is more precise. That too should indeed be done in a subsequent PR.